packages/platforms/accton/x86-64/x86-64-accton-as5912-54xk/onlp/builds/src/module/src/debug.c

```diff
@@ -0,0 +1,45 @@
+#include "x86_64_accton_as5912_54xk_int.h"
```

Comment:
Please remove this file. It doesn't do anything and the debug option has been moved to the SYSI interface.